.github/workflows/go.yml

```diff
@@ -1,125 +1,25 @@
 name: Go
 
-on:
-  push:
-    branches: [main]
-  pull_request:
-    branches: [main]
+on: [push, pull_request]
```

Comment:
This needs to be a map not a list, currently the tests are not running in this PR and they should

Reply:
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/triggering-a-workflow#using-multiple-events

This runs in my fork, I suspect it is because it is a cross-fork PR that the workflows do not run until merged.